### PR TITLE
Adds HPCTM-1459 package

### PR DESCRIPTION
**Pull request by sergiorg-hpc**

### CODE REVIEW

**Review by matz-e — DISMISSED** [review]

**Comment by sergiorg-hpc** [comment]
please retest this

**Comment by sergiorg-hpc** [comment]
retest this please

**Review by matz-e — DISMISSED** [review]

File: deploy/environments/applications.yaml [hunk 1]
```diff
@@ -164,4 +166,4 @@ spack:
     - synapsetool
     - touchdetector
     - ultraliser
-    - unit-test-translator
\ No newline at end of file
+    - unit-test-translator
```
Comment (sergiorg-hpc) [inline deploy/environments/applications.yaml]: Not sure what this is, perhaps some temporary thing someone added back in the day? Not mine, tbh.

Comment (matz-e) [reply deploy/environments/applications.yaml]: Certain people don't have their editor configured right :angry:

**Comment by pramodk** [comment]
@sergiorg-hpc: (a future question) when are you planning to rename and deploy this soon?

**Comment by sergiorg-hpc** [comment]
> @sergiorg-hpc: (a future question) when are you planning to rename and deploy this soon?

Is this expected to be deployed officially on BB5? That is why I kept it as a PR Draft, if you notice.

In any case, during the last meeting it was agreed that I would be in charge of running the tool inside certain important folders to verify that no issues appear. I'm finishing the support for bind paths and this would be my next step. After this and if we do not seem to encounter any problems, we are expected to have a follow-up meeting to coordinate next steps and possibly involve CS.

Regarding the name, it seems to appeal to everybody so far :). Nonetheless, @st4rl3ss suggested `DIRE`, as in DIsaster REcovery.

**Comment by sergiorg-hpc** [comment]
Moved to https://github.com/BlueBrain/spack/pull/1450.

**Pull request closed without merging**